LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Let's remove the license/readme from this PR. Would like to follow guidelines on https://github.com/dashpay/dash/blob/master/CONTRIBUTING.md#contributor-workflow as much as possible